operator/pom.xml

```
@@ -31,8 +31,8 @@
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.operator.sdk.version>6.0.0.CR1</quarkus.operator.sdk.version>
-        <quarkus.version>3.0.0.CR1</quarkus.version>
+        <quarkus.operator.sdk.version>6.0.1</quarkus.operator.sdk.version>
+        <quarkus.version>3.0.1.Final</quarkus.version>
```

Operator SDK 6.0.1 [uses](https://github.com/quarkiverse/quarkus-operator-sdk/blob/6.0.1/pom.xml#L15) Quarkus 3.0.0.  This property is not overriding the Quarkus version that SDK uses. It is just for alignment of other dependencies so it needs to match whatever the SDK uses.
```suggestion
        <quarkus.version>3.0.0.Final</quarkus.version>
```